### DISCUSSION TITLE
望下版本加入系统托盘

### DISCUSSION THREAD

**Discussion by xhrzg2017** [rime/home#1587] [Ideas]
经过部分美化软件后rime设置功能失效
<img width="447" alt="image" src="https://github.com/rime/home/assets/34803905/fb7a6e12-aaca-4889-bd3a-9af9d308431f">

<img width="344" alt="image" src="https://github.com/rime/home/assets/34803905/63a80713-5548-4741-acf5-1b40110f6efb">

**Comment by LEOYoon-Tsaw** [comment]
這是什麼⋯mac？
